script/bootstrap

```diff
@@ -78,4 +78,11 @@ fi
 echo "  Application Bootstrap 2/2. Ansible (allthethings)..."
 ansible-playbook ${playbook} --inventory-file=${inventory} --connection=local
 
-echo "Bootstrap Complete"
\ No newline at end of file
+echo " Install MySQL 5.7 "
+sudo apt-get install mysql-server-5.7
+sudo apt-get install python-mysqldb
+
+echo " Install Virtuel Enviroment"
+sudo pip install virtualenv virtualenvwrapper
```

Comment:
I have a good example for how to deal with the virtual env stuff. Let me see if I can load that up for you tonight.